.github/workflows/ci.yml

```
@@ -0,0 +1,165 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.25.1'
+    
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v3
```

actionlint (1.7.7)

the runner of "golangci/golangci-lint-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)